shared/src/main/kotlin/com/egm/stellio/shared/model/ErrorResponse.kt

```diff
@@ -0,0 +1,19 @@
+package com.egm.stellio.shared.model
```

Comment:
A better approach is to have something like (it is of course to be completed and adjusted) :

```
sealed class ErrorResponse(
    val type: ErrorType,
    val title: String,
    val detail: String
)

data class BadRequestDataException(val detailedMessage: String): ErrorResponse(ErrorType.BAD_REQUEST_DATA, "", detailedMessage)
```

It has the advantages:
- To avoid redundancy and boilerplate code when raising exceptions
- You keep the control of the possible list of exceptions (list that is fixed by NGSI-LD specification)

Reply:
same principle may probably be applied to the exceptions hierarchy